src/Libraries/CoreNodeModelsWpf/CoreNodeModelsWpf - Backup.csproj

```diff
@@ -0,0 +1,155 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I guess that this file should be removed?

Reply:
Yup, absolutely. Removed it.